mmdet/models/detectors/base.py

```diff
@@ -164,6 +164,12 @@ def forward(self, img, img_metas, return_loss=True, **kwargs):
         should be double nested (i.e.  List[Tensor], List[List[dict]]), with
         the outer list indicating test time augmentations.
         """
+        img_norm_cfg = kwargs.get('img_norm_cfg', None)
+        if img_norm_cfg:
+            mean = torch.tensor(img_norm_cfg['mean'])[None, ..., None, None]
+            std = torch.tensor(img_norm_cfg['std'])[None, ..., None, None]
```

Comment:
We suggest keeping this logic in lines between 173-175 to restrict its influence.